apps/aehttp/src/ws_int_dispatch.erl

```diff
@@ -28,7 +28,7 @@ is_valid(_, _, _) ->
     true.
 
 do_execute(chain, get, QueryPayload) ->
-    #{<<"type">> := Type} = QueryPayload,
+    Type = maps:get(<<"type">>, QueryPayload, <<"block">>),
```

Comment:
Why a default? I would rather expect to keep this code simpler not having the default, documenting what is needed (in protocol repo) and requiring client to comply to documented schema.
(I may not understand the problem here so please correct me if wrong.)